spec/spec_helper.rb

```diff
@@ -1,3 +1,9 @@
-require 'coveralls'
+require 'coveralls'  
 Coveralls.wear!
+
+# couldn't get to work, error: Nokogiri was built against LibXML version 2.9.3, but has dynamically loaded 2.9.2  T
```

Comment:
Metrics/LineLength: Line is too long. [115/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)